File: 프로그래머스/2/12914. 멀리 뛰기/멀리 뛰기.py
def solution(n):
    BIG = 1234567
    
    if n == 1:
        return 1
    
    answer = [0] * n
    
    answer[0] = 1
    answer[1] = 2
    
    for i in range(2, n):
        answer[i] = answer[i - 1] + answer[i - 2]
    
    return answer[n - 1] % BIG

File: 프로그래머스/2/12914. 멀리 뛰기/멀리 뛰기.js
function solution(n) {
    const MOD = 1234567;
    
    let dp = [];
    
    dp[1] = 1;
    dp[2] = 2;
    
    for (let i = 3; i <= n; i++) {
        dp[i] = (dp[i - 1] + dp[i - 2]) % MOD;
    }
    
    return dp[n];
}